lib/schwifty-migrate-diff.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+// Allowing path to be set for testing
+exports.get = (path) => {
+
+    path = path || 'schwifty-migrate-diff';
+
+    let SchwiftyMigrateDiff;
+    try {
+        SchwiftyMigrateDiff = require(path);
+    }
+    catch (err) {
```

Comment:
This `err` looks like it's ignored.  You might want to do something similar to haute-couture: https://github.com/hapipal/haute-couture/blob/master/lib/index.js#L28

Reply:
Handling it below (Just commenting to ensure it's noted)